loopchain/blockchain/blockchain.py

```diff
@@ -705,7 +708,8 @@ def confirm_prev_block(self, current_block: Block):
 
             # utils.logger.debug(f"-------------------confirm_prev_block---before add block,"
             #                    f"height({unconfirmed_block.header.height})")
-            self.add_block(unconfirmed_block, current_block.body.confirm_prev_block)
+            confirm_info = current_block.body.prev_votes if current_block.header.version == "0.3" else None
```

Comment:
Is the value of `None` fine for other versions to except for v0.3?
I wonder why you don't use `current_block.body.confirm_prev_block` for other versions.